bin/install-wp-tests.sh

```
@@ -0,0 +1,168 @@
+DB_USER=$2
+DB_PASS=$3
+DB_HOST=${4-localhost}
+WP_VERSION=${5-latest}
+SKIP_DB_CREATE=${6-false}
+MULTISITE=${7-false}
+
+WP_TESTS_DIR=${WP_TESTS_DIR-/tmp/wordpress-tests-lib}
+WP_CORE_DIR=${WP_CORE_DIR-/tmp/wordpress/}
+
+download() {
+    if [ $(which curl) ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)